### PR TITLE
Experiment using Jetpack Logo package in VaultPress

### PR DESCRIPTION
**Pull request by gititon**
This PR experiments with adding the Jetpack logo to VaultPress via the Jetpack Logo package. 

Clone this branch into your local Jetpack environment's plugins folder. Run `composer update`. Go to Jetpack in wp-admin, VaultPress. You should see the Jetpack logo as depicted in the screenshot below if you haven't yet added a subscription.

Screen Shot 2019-05-23 at 6 38 11 PM

### CODE REVIEW

**Comment by gititon** [comment]
Thanks @tyxla. To be sure, I had it working with the original autoloader as well and had addressed the other comments this PR received in my local branch. I was trying to create a package that would eliminate the need to duplicate lines 17 - 43 from having to be included in every Jetpack ecosystem package. I will move on from it to something else since your changes landed first. 👍

**Comment by tyxla** [comment]
> I was trying to create a package that would eliminate the need to duplicate lines 17 - 43 from having to be included in every Jetpack ecosystem package.

Well, that will still be useful 😉 As long as it saves us from duplicating the same stuff, it sounds like a good idea!

**Comment by tyxla** [comment]
Update: Just updated the PR so it uses the Packagist logo package 👍

File: vaultpress.php [hunk 1]
```diff
@@ -14,6 +14,32 @@
 // don't call the file directly
 defined( 'ABSPATH' ) or die();
 
+/**
+ * Load all the packages.
+ *
+ * We want to fail gracefully if `composer install` has not been executed yet, so we are checking for the autoloader.
+ * If the autoloader is not present, let's log the failure, pause VaultPress, and display a nice admin notice.
+ */
+$jetpack_autoloader = dirname( __FILE__ ) . '/vendor/autoload.php';
```
Comment (gravityrail) [inline vaultpress.php]: this is not a "jetpack_autoloader". It's just an autoloader. You can call this var `$loader` or `$autoloader` if you like.

File: vaultpress.php [hunk 2]
```diff
@@ -651,6 +680,14 @@ function ui_delete_vp_settings_button() {
 		<?php
 	}
 
+	function ui_logo() {
+		?>
+		<div class="clearfix"></div>
+		<?php
+		$logo = new Automattic\Jetpack\Assets\Logo();
```
Comment (gravityrail) [inline vaultpress.php]: Isn't it customary for a namespace reference to have a leading slash? Like `new \Automattic\Jetpack\Assets\Logo()`?

**Comment by eliorivero** [comment]
This is awesome. This can be merged and I'll use it in the UI restyling

https://github.com/Automattic/vaultpress/pull/6

File: vaultpress.php (hunk 2 above)
Comment (lezama) [reply vaultpress.php]: https://stackoverflow.com/questions/4849279/importing-classes-and-namespaces-in-php-what-difference-does-a-leading-backslas

not sure what's the conclusion 😅

Comment (tyxla) [reply vaultpress.php]: AFAIK, leading backslashes aren't necessary. In addition to the post @lezama linked, here's some evidence from the PHP documentation to back that up:

> Note that for namespaced names (fully qualified namespace names containing namespace separator, such as Foo\Bar as opposed to global names that do not, such as FooBar), the leading backslash is unnecessary and not recommended, as import names must be fully qualified, and are not processed relative to the current namespace.

Source: https://php.net/manual/en/language.namespaces.importing.php#language.namespaces.importing

File: vaultpress.php (hunk 1 above)
Comment (tyxla) [reply vaultpress.php]: Thanks for this! I was planning to clean that up, but never did 😅 (Miguel fixed it in https://github.com/Automattic/vaultpress/pull/5/commits/76c46134dfd7453b49fc0cf2a2641ec332c8526d)

**Comment by lezama** [comment]
@tyxla we can assume that there will be no release before https://github.com/Automattic/vaultpress/pull/6. @eliorivero is going to take care of the design side :)

**Pull request merged**